common/src/main/java/io/netty/util/concurrent/DefaultRejectTaskHandler.java

```diff
@@ -0,0 +1,18 @@
+package io.netty.util.concurrent;
+
+import io.netty.util.internal.logging.InternalLogger;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+
+/**
+ * The default implementation just logs the rejection of a rejected task. 
+ * 
+ */
+public class DefaultRejectTaskHandler implements RejectTaskHandler {
```

Comment:
Make it final, add a private constructor and add a public static final instance field ? There is not need to create a new instance of this overtime.